docs/api/app.md

```diff
@@ -1252,6 +1252,26 @@ Returns `String` - The command-line switch value.
 
 **Note:** When the switch is not present or has no value, it returns empty string.
 
+### `app.enableSecureMode()`
+
+Changes the default values of following `webPreferences`:
```

Comment:
In anticipation of this API doing more things in the future this should probably be moved to `api/secure-mode.md` or similar so we can add detail / more things to it without making the app documentation page huge.

Reply:
I was thinking about that, makes sense. I will do it.